source/codegen/metadata/nifake_non_ivi/config.py

```diff
@@ -35,13 +35,14 @@
     'additional_headers': ['custom/nidaqmx_conversions.h'],
     'type_to_grpc_type': {
         'FakeHandle': 'nidevice_grpc.Session',
-        'const char[]': 'string',
+        'char[]': 'string',
```

Comment:
Having to handle `const` and non-const separately adds an extra lookup condition to get grpc_type conversion method. But we always get the same result. So standardizing on non-const here and the conversion method will strip const before the lookup.